bundles/binding/org.openhab.binding.smarthomatic/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,35 @@
+Manifest-Version: 1.0
+Private-Package: org.openhab.binding.smarthomatic.internal
+Ignore-Package: org.openhab.binding.smarthomatic.internal
+Bundle-License: http://www.eclipse.org/legal/epl-v10.html
+Bundle-Name: openHAB Smarthomatic Binding
+Bundle-SymbolicName: org.openhab.binding.smarthomatic
+Bundle-Vendor: openHAB.org
+Bundle-Version: 0.1.0.qualifier
+Bundle-Activator: org.openhab.binding.smarthomatic.internal.SmarthomaticActivator
+Bundle-ManifestVersion: 2
+Bundle-Description: This is the Smarthomatic binding of the open Home Aut
+ omation Bus (openHAB)
+Import-Package: gnu.io,
+ org.apache.commons.io;version="2.0.1",
```

Comment:
If this specific version of commons-io is required, please document it in about.html. Else, remove the version restriction.